### ISSUE TITLE
Unable to use squashfs without tar layers

### ISSUE THREAD

**Issue by smoser** [project-stacker/stacker#319]
I'm trying to use only squashfs layers, and running into a problem that occurs both as unprivileged and privileged user.

Basically trying to build this stacker.yaml with only `--layer-type=squashfs`
```yaml
rootfs:
  from:
    type: docker
    url: "${{BASE_URL}}${{LAYER_SUFFIX}}"
  import:
    - hello-world
  run: |
    cp /stacker/hello-world /usr/bin/
    chmod 755 /usr/bin/hello-world
    /usr/bin/hello-world
```

This can be reproduced with [go-test.sh](https://github.com/project-stacker/stacker/files/9842697/go-test.sh.txt) as root or non-root.  Just point it at a url that has both -squashfs and tar formats.  Both root and non-root paths fail.

Output of go-test.sh is attached:
 * as root : [root.log](https://github.com/project-stacker/stacker/files/9842702/root.log)
 * as non-root [no-root.log](https://github.com/project-stacker/stacker/files/9842700/no-root.log)

**Comment by smoser** [comment]
I've just now tested with stacker 0.3.1 and it fails as both root and non-root.

[root-0.3.1.log.txt](https://github.com/project-stacker/stacker/files/10127562/root-0.3.1.log.txt) `sudo rm -Rf sdir-*; sudo PATH=/tmp/stackers/0.30.1:$PATH ./go-test2.sh  2>&1 | tee no-root-0.3.1.log`
[no-root-0.3.1.log.txt](https://github.com/project-stacker/stacker/files/10127563/no-root-0.3.1.log.txt) `sudo rm -Rf sdir-*; PATH=/tmp/stackers/0.30.1:$PATH ./go-test2.sh  2>&1 | tee no-root-0.3.1.log`

The one change to the script is setting the baseurl (to something built with a recent stacker).


#329 fixes this issue.

See (v0.22.1-8-gb287f83)
[root-dev.log](https://github.com/project-stacker/stacker/files/10127583/root-dev.log) - `sudo rm -Rf sdir-*; sudo PATH=/tmp/stackers/dev:$PATH ./go-test2.sh  2>&1 | tee /tmp/root-dev.log`
[no-root-dev.log](https://github.com/project-stacker/stacker/files/10127584/no-root-dev.log) - `sudo rm -Rf sdir-*; PATH=/tmp/stackers/dev:$PATH ./go-test2.sh  2>&1 | tee /tmp/no-root-dev.log`

**Comment by smoser** [comment]
This was fixed by #329 (i think).
I just tested the script now with  v1.0.0-rc5-396ff9d and it does not fail.
